pkg/tinydtls/include/sock_dtls_types.h

```diff
@@ -34,6 +34,13 @@ extern "C" {
 #define SOCK_DTLS_MBOX_SIZE     (4)         /**< Size of DTLS sock mailbox */
 #endif
 
+/**
+ * @brief Information about remote client connected to the server
+ */
+struct sock_dtls_session {
+    session_t       dtls_session;    /**< TinyDTLS session */
+};
```

Comment:
Why move this?

Reply:
Oh, just a leftover from another approach I tried. Will fix!